components/layout.tsx

```diff
@@ -80,20 +80,18 @@ const TopNav: React.FC<React.PropsWithChildren<unknown>> = () => {
   const { authenticated, claims } = useAuth()
   const { profile } = useProfile()
   const isMobile = useMediaQuery("(max-width: 768px)")
-  const [sticky, setSticky] = useState(false)
+  const [sticky, setSticky] = useState("top")
```

Comment:
nit: Do we actually still use this `sticky` state, or can we remove it?

Reply:
we can remove it; I caught it the code I'm putting towards the next PR that follows this one.